docker/dockerfiles/polkadot/polkadot_builder.Dockerfile

```
@@ -1,5 +1,5 @@
 # This is the build stage for Polkadot. Here we create the binary in a temporary image.
-FROM docker.io/paritytech/ci-linux:production as builder
+FROM docker.io/paritytech/ci-unified:latest as builder
```

```suggestion
FROM docker.io/paritytech/ci-unified:bullseye-1.77.0-2024-04-10-v20240408 as builder
```
I'd use current image (can be found [here](https://github.com/paritytech/polkadot-sdk/blob/master/.github/env)) because latest can be newer